package.json

```diff
@@ -113,7 +114,7 @@
     "@babel/polyfill": "^7.8.7",
     "@babel/runtime": "^7.9.6",
     "@patternfly/patternfly": "^2.71.5",
-    "@patternfly/react-core": "^3.153.13",
```

Comment:
We have to use at least version `3.153.7` due to this bug fix https://github.com/patternfly/patternfly-react/pull/3990 I think we can be fine with the extra bundle size for now.

Reply:
:cry: ok I will put It back. Hopefully they will fix it soon.

Reply:
Ok i used the latest version of PF. That added around 0.5MB to the inventory frontend

Reply:
If it's in inventory frontend then it's no big deal.